src/signalrclient/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ else()
     )
   elseif(NOT WIN32)
     target_link_libraries(signalrclient
-      PUBLIC ${CPPREST_LIB}
+      PUBLIC ${CPPREST_LIB} Boost::system
```

Comment:
Do you not need `find_package(Boost REQUIRED COMPONENTS system)` anymore?

Reply:
You have declared `find_package(Boost REQUIRED COMPONENTS system thread chrono)` in `./CMakeLists.txt` (line 43).